File: Level 36/Homework/HW9.py
# 9) შექმენი ფუქნცია რომელიც მიიღებს რიცხვების სიას და დააბრუნებს მხოლოდ ლუწ რიცხვებს

def even(numbers):
    result = []
    for num in numbers:
        if num % 2 == 0:
            result.append(num)
    return result

nums = [64, 52, 3, 69, 5, 44]
print(even(nums))

File: Level 100/Homework/HW9.js
const numbers = [4, 8, 12, 25, 30, 40, 50]

function numberjumber(numbers){
    let i = 0
    while(i < numbers.length){
        if(numbers[i] > 20){
            console.log(numbers[i])
            break
        }

        i++
    }
}

numberjumber(numbers)